substrate/frame/tx-pause/src/mock.rs

```diff
@@ -21,13 +21,10 @@
 
 use super::*;
 use crate as pallet_tx_pause;
-
-use frame_support::{
-	derive_impl, parameter_types,
-	traits::{ConstU64, Everything, InsideBoth, InstanceFilter},
+use frame::{
+	testing_prelude::*,
+	traits::{Everything, InsideBoth, InstanceFilter},
```

Comment:
It seems that `InsideBoth` and `InstanceFilter` are used in more than one pallet (safe-mode pallet), so you should add them to the testing prelude.